File: ModuleLib/mvpbase-appcompat/src/main/java/com/baozi/mvp/model/PagerModel.java
package com.baozi.mvp.model;

import android.support.annotation.DrawableRes;
import android.view.View;

import java.util.List;

/**
 * Created by Administrator on 2017/8/16 0016.
 */

public interface PagerModel {
    Class[] getFragments();

    String[] getTabString();

    @DrawableRes
    int[] getTabDrawables();

    boolean isAnimation();
}

File: lib-mvpbase-kotlin/src/main/java/com/linfeng/mvp/model/PagerModel.kt
package com.linfeng.mvp.model

import android.support.annotation.DrawableRes

/**
 * Created by Administrator on 2017/8/16 0016.
 */

interface PagerModel {

    val fragments: Array<Class<*>>

    val tabString: Array<String>

    @get:DrawableRes
    val tabDrawables: IntArray

    val isAnimation: Boolean
}
